lib/migration.pm

```diff
@@ -39,8 +39,12 @@ sub setup_migration {
     select_console 'root-console';
 
     # stop packagekit service
-    systemctl 'mask packagekit.service';
-    systemctl 'stop packagekit.service';
+    # Systemd is not available on SLE11
+    # skip this part if the version below SLE12
```

Comment:
sle11 doesn't have systemd, but packagekitd is still run on it. I suggest to execute this command to stop it on sle11:
`assert_script_run "chmod 444 /usr/sbin/packagekitd"`

Reply:
Verification run: http://10.67.17.147/tests/2138#step/patch_before_migration/8